samples/v1.6/Tests/Carousel.ForbiddenActions.json

```diff
@@ -2,194 +2,201 @@
 	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
 	"type": "AdaptiveCard",
 	"version": "1.6",
-	"display": {
-		"type": "carousel",
-		"timer": 5000
-	},
-	"body": [
-		{
-			"type": "CarouselPage",
-			"id": "firstCarouselPage",
-			"items": [
-				{
-					"type": "Container",
-					"items": [
-						{
-							"type": "TextBlock",
-							"text": "There should be no actions rendered below",
-							"wrap": true,
-							"id": "id1"
-						}
-					]
-				}
-			]
-		},
-		{
-			"type": "CarouselPage",
-			"id": "secondCarouselPage",
-			"items": [
-				{
-					"type": "RichTextBlock",
-					"inlines": [
-						"No action should be rendered here ",
-						{
-							"type": "TextRun",
-							"text": "this is a showcard action,",
-							"selectAction": {
-								"type": "Action.ShowCard",
-								"title": "Action.ShowCard",
-								"card": {
-									"type": "AdaptiveCard",
-									"body": [
-									{
-										"type": "TextBlock",
-										"text": "What do you think?"
-									}
-									],
-									"actions": [
-									{
-										"type": "Action.Submit",
-										"title": "Neat!"
+	"body": {
+		"type": "Carousel",
+		"pages": [
```

Comment:
I prefer CarouselItems, CarouselSet or CarouselPages.

Reply:
we can discuss `items` vs. `pages`, but all of our existing container-like elements have their collections in a property with a single word all-lowercase name (e.g. `items` or `columns`)